tests/models/test_pose.py

```diff
@@ -0,0 +1,8 @@
+from alitra.models import Pose
+
+
+def test_pose(default_position, default_orientation, robot_frame):
+    pose: Pose = Pose(
+        position=default_position, orientation=default_orientation, frame=robot_frame
+    )
+    assert pose.frame == robot_frame
```

Comment:
In my opinion, this test does not give much. It tests that the creation of a dataclass work as intended. Since we have not implemented the dataclass in this library, I would say this test is redundant.

If we still are to have the test, why not test all three attributes of the class (position and orientation as well)?

Reply:
Restructured the test, hopefully it makes more sense now